icecreamshop/icecreamshop/src/main/java/com/summerschool/icecreamshop/model/Product.java

```diff
@@ -1,31 +1,43 @@
 package com.summerschool.icecreamshop.model;
 
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotNull;
 import javax.persistence.*;
 import java.util.List;
 
 @Entity
 public class Product {
     @Id
```

Comment:
Please leave an empty line above each annotation. If you have multiple annotations on a single attribute, leave the empty line above the first annotation. Good practice tip